feature-libs/product/future-stock/components/future-stock-accordion/future-stock-accordion.component.html

```diff
@@ -0,0 +1,27 @@
+<button
+  class="cx-future-stock-accordion-header"
+  [attr.aria-expanded]="expanded"
```

Comment:
this would break toggle behavior, you would need to subscribe to behaviorsubject in the component

Reply:
I believe we could achieve this with an `ng-container *ngIf="expanded$ | async as expanded"` enclosing the component so that nothing else needs to change in the template.